### DISCUSSION TITLE
Question regarding authentication methods for outlook inbox in Azure.

### DISCUSSION THREAD

**Discussion by rinzler-flynn** [nautobot/nautobot-plugin-circuit-maintenance#211] [Q&A] [answered]
For an outlook inbox specifically, I have the following requirement to use OAUTH to access a shared email inbox via o365. Does the plugin support provider configuration to support this? Thanks.

**Comment by chadell** [comment, marked as answer]
OAUTH for o365 is not currently implemented, only IMAP and Gmail OAUTH are.

The o365 OAUTH would be something similar to https://github.com/nautobot/nautobot-plugin-circuit-maintenance/blob/develop/nautobot_circuit_maintenance/handle_notifications/sources.py#L578
